scripts/CeruleanGym.asm

```diff
@@ -185,23 +185,23 @@ CeruleanGymAfterBattleText2:
 	text_far _CeruleanGymAfterBattleText2
 	text_end
 
-CeruleanGymText4:
+CeruleanGymFanText:
```

Comment:
I think what you're calling the "gym fan" is called "gym guide" elsewhere. Both are okay, but I think I prefer guide.